consensus/src/pipeline/buffer_manager.rs

```diff
@@ -708,7 +708,7 @@ impl BufferManager {
                 // find the corresponding item
                 let author = vote.author();
                 let commit_info = vote.commit_info().clone();
-                info!("Receive commit vote {} from {}", commit_info, author);
+                debug!("Receive commit vote {} from {}", commit_info, author);
```

Comment:
I would put this one as a trace , if we don't need it. it's already filtered out from k8s, because it is so spammy